hazelcast/src/main/java/com/hazelcast/internal/management/events/ConfigUpdateFailedEvent.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management.events;
+
+import com.hazelcast.internal.config.ConfigNamespace;
+import com.hazelcast.internal.json.JsonObject;
+
+import java.util.UUID;
+
+public class ConfigUpdateFailedEvent extends AbstractIdentifiedEvent {
+    private final Exception exception;
```

Comment:
could you please add information about the added configs to `ConfigUpdateFailedEvent` as it's present `ConfigUpdateFinishedEvent`?

Reply:
https://github.com/hazelcast/hazelcast/pull/20260/commits/31eb22ebc8da0d0fe120bfb5b79b10558f72e4b3